### PR TITLE
Announce Tenzir v4.17

### PR DESCRIPTION
**Pull request by dominiklohmann**
The usual.

### CODE REVIEW

File: web/blog/tenzir-v4.17/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+---
+title: Tenzir v4.17
+authors: [dominiklohmann]
+date: 2024-06-21
+tags: [release, azure-log-analytics, lookup-table]
+comments: true
+---
+
+The new [Tenzir v4.17][github-release] brings an integration with Azure Log
+Analytics and introduces two mechanisms for expiring entries in lookup tables.
```
Comment (mavam) [inline web/blog/tenzir-v4.17/index.md:10]:
```suggestion
Analytics and adds support for expiring entries in lookup tables.
```

File: web/blog/tenzir-v4.17/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+`--update-timeout <duration>`.
+
+Both new options cause individual events to expire in the lookup table. Create
+timeouts specify the time after which entries in the lookup table expire, and
+update timeouts specify the time after which entries in the lookup table expire
+when they're not accessed.
+
+```text {0} title="Expire lookup table entries after a week, or when not accessed in a day"
+…
+| context update my-lookup-table --create-timeout 1w --update-timeout 1d
```
Comment (mavam) [inline web/blog/tenzir-v4.17/index.md:46]: I think we need to be very clear that the minimum timeout is winning here. Saying "or" is not very clear.

**Review by mavam — APPROVED** [review]
Reads smoothly!

File: web/blog/tenzir-v4.17/index.md (hunk 2 above)
Comment (dominiklohmann) [reply web/blog/tenzir-v4.17/index.md:46]: Do you think a "…, whichever expires first" helps?

File: web/blog/tenzir-v4.17/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+schema inference is the modus operandi. This caused data that was imported with
+a schema to sometimes use a `timestamp` type, but all automatically inferred
+data used the underlying `time` type. This caused issues down the line, because
+operators like `summarize` by design to not group fields together with distinct
+types. To users, this showed as duplicate values that were supposed to be
+grouped by in summarized results.
+
+:::warning Required Configuration Changes
+If you have custom schemas installed in `/opt/tenzir/etc/tenzir/schemas` or
+`~/.config/tenzir/schemas`, you will need to adaot them in one of two ways:
```
Comment (dominiklohmann) [inline web/blog/tenzir-v4.17/index.md:105]:
```suggestion
`~/.config/tenzir/schemas`, you will need to adapt them in one of two ways:
```

**Pull request merged**